docs/guide/alerts-guide.md

```diff
@@ -0,0 +1,95 @@
+
+Sematext Alerts can cover both complicated alerting scenarios with multiple rules on both Logs and Monitoring, as well as Events. A basic Alert Policy with an email or Slack notification is enough to cover elementary alerting needs.
+
+### Alerting on Logs
+
+To **create an Alert** you need to enter a **search query** and press **Save Query/Alert Rule**. 
+
+![Create Alert Search Query](../images/guide/alerts-and-events/create-alert-logs-search-query.png)
+
+A **Save Query** window will open with the option to **Enable Alert**. When it's toggled, the **Alert Type picker** and **Condition** will expand, and the **Notifications** and **Schedule** tabs will be enabled. You'll see:
+
+- **Alert Type**, to choose either Threshold Alerts or Anomaly Alerts. For more information on alerting capabilities check [alerts](https://sematext.com/docs/alerts/) documentation pages.
+- **Condition**, to choose when the Alert will be triggered.
+
+![Enable Alert Notification](../images/guide/alerts-and-events/save-alert-1.png)
+
+- **Notification hook details** with a **default email** notification hook used to send the message to your account. You can also add additional email addresses as well as other types of notification hooks.
+
+![Set Alert Notification Hooks](../images/guide/alerts-and-events/save-alert-2.png)
+
+- **Schedule**, to choose when the Alerts will notify you based on a period, schedule or time.
+
+![Set Alert Schedule](../images/guide/alerts-and-events/save-alert-3.png)
+
+Press Save and you're done. Check out the [integrations](https://sematext.com/docs/integration/) documentation for more information about alert notification hooks.
+
+### Alerting on Metrics
+
+Metrics have both **Threshold** and **Anomaly** Alerts, but a Monitoring App has **component-based Alerting**. Every time-series metric is a _component_. Alerts can be created for each component, just as they can be added to custom [dashboards](./dashboards-guide/).
+
+![Sematext Cloud Metric Component Alerts](https://sematext.com/docs/images/guide/alerts-and-events/metric-component-alert.png "Sematext Cloud Metric Component Alerts")
+
+Monitoring Apps also have **Heartbeat** Alerts as part of their settings. You can find it in the App Settings dropdown menu. They are triggered when what you are monitoring, like your servers, containers, or your applications stop sending data to Sematext.
+
+![Sematext Cloud Metric Component Custom Alerts](https://sematext.com/docs/images/guide/alerts-and-events/create-heartbeat-alerts.png "Sematext Cloud Metric Component Custom Alerts")
+
+## Next step
+
+[Sign up](https://apps.sematext.com/ui/registration) to Sematext Cloud for free to get started, and create your first Logs App. Upload logs from all your servers to our centralized log management solution through our Elasticsearch API, and experience a proper hosted Elasticsearch Stack.
+
+If you already have an account, [sign in](https://apps.sematext.com/ui/login/) and start shipping logs!
```

Comment:
Do we need this and the above paragraph in the docs?

Reply:
Removed.